xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-api/src/main/java/org/xwiki/icon/IconSet.java

```diff
@@ -171,13 +182,54 @@ public String getRenderHTML()
 
     /**
      * Set the HTML code (containing velocity) to display an icon from this set.
+     *
      * @param renderHTML the HTML code to set
      */
     public void setRenderHTML(String renderHTML)
     {
         this.renderHTML = renderHTML;
     }
 
+    /**
+     * @return the icon url
+     * @since 10.6RC1
+     */
+    public String getUrl()
```

Comment:
``iconSet.getURL()`` makes me think it returns the **icon set URL** not the **icon URL**. I would use a more explicit name, such as ``iconSet.getIconURL()``.